wp-content/themes/newspack-theme/acf-json/group_622a0f70c890f.json

```diff
@@ -0,0 +1,46 @@
+{
+    "key": "group_622a0f70c890f",
+    "title": "Episode Audio",
+    "fields": [
+        {
+            "key": "field_622a102fdd5e8",
+            "label": "Audio",
+            "name": "audio",
+            "type": "clone",
```

Comment:
ACF has an option to reference an already existing field in the db. Did that here for audio and broadcast date.